configs/sngan_proj/README.md

```diff
@@ -22,15 +22,14 @@
+| sngan_proj_128 | ImageNet |      w       |   5    |    1000000    | [28.1799 (944000)](https://download.openmmlab.com/mmgen/sngan_proj/sngan_proj_imagenet1k_128_Glr2e-4_Dlr5e-5_ndisc5_b128x2_wReLUinplace_is-iter944000_20210730_132714-ca0ccd07.pth)  | [33.4821 (988000)](https://download.openmmlab.com/mmgen/sngan_proj/sngan_proj_imagenet1k_128_Glr2e-4_Dlr5e-5_ndisc5_b128x2_wReLUinplace_fid-iter988000_20210730_132401-9a682411.pth)  | [config](https://github.com/open-mmlab/mmgeneration/tree/master/configs/sngan_proj/sngan_proj_128_wReLUinplace_imagenet1k_b128x2_Glr-2e-4_Dlr-5e-5_ndisc5.py)  | [Log](https://download.openmmlab.com/mmgen/sngan_proj/sngan_proj_imagenet1k_128_Glr2e-4_Dlr5e-5_ndisc5_b128x2_wReLUinplace_20210730_132401_fid-9a682411_is-ca0ccd07.json)  |
 | sngan_proj_128 | ImageNet |     w/o      |   2    |    2000000    |                                                                                     Coming soon                                                                                      |                                                                                     Comming soon                                                                                      |                                                                          Comming Soon                                                                          |                                                                                Comming Soon                                                                                |
 | sngan_proj_128 | ImageNet |      w       |   2    |    2000000    |                                                                                     Coming soon                                                                                      |                                                                                     Comming soon                                                                                      |                                                                          Comming Soon                                                                          |                                                                                Comming Soon                                                                                |
 
 '\*' Iteration counting rule in our implementation is different from others. If you want to align with other codebases, you can use the following conversion formula:
```

Comment:
Here, you may use `disc_steps`  in line with our configs rather than `n_disc`. You should also clarify the meaning of `n_disc` and `disc_steps`.